eng/versioning/external_dependencies.txt

```diff
@@ -91,11 +91,11 @@ org.postgresql:postgresql;42.3.9
 org.slf4j:slf4j-api;1.7.36
 org.slf4j:slf4j-nop;1.7.36
 org.slf4j:slf4j-simple;1.7.36
-redis.clients:jedis;5.2.0
-io.lettuce:lettuce-core;6.7.1.RELEASE
-org.redisson:redisson;3.50.0
-testdep_net.bytebuddy:byte-buddy;1.17.5
-testdep_net.bytebuddy:byte-buddy-agent;1.17.5
+redis.clients:jedis;6.2.0
```

Comment:
Are there any breaking changes due to the major version update of this dependency? Is this only used in the Event Hubs checkpoint store which isn't GA yet?

Reply:
Yeah, this is only used in a non-GA library.